tutorials/filter/src/main/java/com/datatorrent/tutorial/filter/Application.java

```diff
@@ -0,0 +1,44 @@
+package com.datatorrent.tutorial.filter;
+
+import org.apache.apex.malhar.lib.fs.FSRecordReaderModule;
+import org.apache.apex.malhar.lib.fs.GenericFileOutputOperator.StringFileOutputOperator;
+import org.apache.hadoop.conf.Configuration;
+
+import com.datatorrent.api.DAG;
+import com.datatorrent.api.StreamingApplication;
+import com.datatorrent.api.annotation.ApplicationAnnotation;
+import com.datatorrent.contrib.formatter.CsvFormatter;
+import com.datatorrent.contrib.parser.CsvParser;
+import com.datatorrent.lib.filter.FilterOperator;
+
+/**
+ * Simple application illustrating file input-output
+ */
+@ApplicationAnnotation(name="FilterExample")
+public class Application implements StreamingApplication
+{
+
+  @Override
+  public void populateDAG(DAG dag, Configuration conf)
+  {
+    
+    FSRecordReaderModule recordReader = dag.addModule("recordReader", FSRecordReaderModule.class);
+    CsvParser csvParser = dag.addOperator("csvParser", CsvParser.class);
+    FilterOperator filterOperator = dag.addOperator("filterOperator", new FilterOperator());
+    
+    CsvFormatter selectedFormatter = dag.addOperator("selectedFormatter", new CsvFormatter());
```

Comment:
Not possible since we want to compare output with input. Also, example should be close to what end user will need his own app. CSV Parser, CSV formatter are inevitable components for any operator processing POJO. Thus, using these operators in the example app is justified.